extensions/net/Cargo.toml

```diff
@@ -23,6 +23,7 @@ http = "0.2.3"
 hyper = { version = "0.14.9", features = ["server", "stream", "http1", "http2", "runtime"] }
 rustls = "0.19.0"
 serde = { version = "1.0.125", features = ["derive"] }
+serde_with = "1.9.3"
```

Comment:
`serde_with` used for deserialization to store a string or a list of string as a `vec`

Reply:
it will be removed